### PR TITLE
Bump bm cihosts to EL9

### PR DESCRIPTION
**Pull request by derekhiggins**

### CODE REVIEW

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

I've updated the dev-scrips PR with 
+echo 0 | sudo dd of=/proc/sys/net/ipv6/conf/ostestbm/addr_gen_mode
+echo 0 | sudo dd of=/proc/sys/net/ipv6/conf/ostestbm/autoconf

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-serial-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

File: ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh [hunk 1]
```diff
@@ -339,6 +343,9 @@ sysctl -w net.ipv6.conf.\$(ip -o route get 1.1.1.1 | cut -f 5 -d ' ').accept_ra=
 
 cd dev-scripts
 
+git fetch https://github.com/derekhiggins/dev-scripts cs9
```
Comment (andfasano) [inline ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh:346]: just for testing?

**Comment by andfasano** [comment]
/pj-rehearse pull-ci-openshift-metal3-dev-scripts-master-e2e-agent-compact-ipv4

File: ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh (hunk 1 above)
Comment (derekhiggins) [reply ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh:346]: yup, I'll remove it once this merges https://github.com/openshift-metal3/dev-scripts/pull/1664

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Comment by derekhiggins** [comment]
/hold 
so that we can alter pool sizes at the same time as this merges

**Comment by derekhiggins** [comment]
/pj-rehearse periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-ovn-serial-virtualmedia periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ipi-upgrade-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv4 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-ovn-ipv6 periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-sdn-bm periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.15-e2e-metal-ipi-serial-ovn-ipv6 pull-ci-openshift-machine-api-operator-release-4.15-e2e-metal-ipi-ovn-dualstack periodic-ci-openshift-release-master-nightly-4.16-e2e-metal-ipi-ovn-ipv6

**Pull request merged**